students/johnpharmd/lesson08/circle_class.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+
+class Circle:
+    """represents a simple circle, using properties, a class method, and
+    special methods"""
+
+    @classmethod
```

Comment:
It would be helpful to add a docstring explaining what the purpose of this method is.